demo-v1/src/hooks/useCti.ts

```diff
@@ -1,25 +1,127 @@
 import { useMemo, useState } from "react";
 
+// import CallingExtensions, { Constants } from "@hubspot/calling-extensions-sdk";
 // @ts-expect-error module not typed
-// import CallingExtensions from "@hubspot/calling-extensions-sdk";
 import CallingExtensions from "../../../src/CallingExtensions";
+// @ts-expect-error module not typed
+import { messageType } from "../../../src/Constants";
+
+const logMsg = (event: any, prefix: string) => {
+  if (event.data) {
```

Comment:
![Screenshot 2023-02-23 at 19 44 30](https://user-images.githubusercontent.com/11711699/221073232-ca624351-0f7b-46b0-b1a2-9dca869500ca.png)

I'm seeing events getting logged twice, which confused me at first. I now understand that the SDK is logging events [through the iFrameManager](https://github.com/HubSpot/calling-extensions-sdk/blob/master/src/IFrameManager.js#L259) in debug mode. I now understand why you were thinking this story may better fit the Message Acknowledgement epic. Long term, I think we should update the way logging is happening in the iFrameManager, but for now, what do you think about logging without the `event.data`, i.e. `[To HubSpot] OUTGOING_CALL_STARTED`, before we call `super.outgoingCall(callDetails)`? This way we add more context without the duplication.

Reply:
We could also go ahead and update the iFrameManager in this ticket since we are going to be increasing the minor version anyways to release the `project-demo-v1` branch.

Reply:
That makes sense - I'll update the iFrameManager since we're releasing the minor version